wagtaildraftail/client/wagtaildraftail.js

```diff
@@ -6,11 +6,20 @@ import DraftailEditor from 'draftail';
 import 'draftail/dist/draftail.css';
 import './wagtaildraftail.css';
 
-import sources from './sources';
 import decorators from './decorators';
+import sources from './sources';
+import {registerDecorator, getDecorator, registerSource, getSource, registerStrategy, getStrategy} from './registry';
+
+// Register Decorators, Sources and Strategies
+Object.keys(decorators).forEach(function(key) {
+  registerDecorator(key, decorators[key]);
+});
+Object.keys(sources).forEach(function(key) {
+  registerSource(key, sources[key]);
+});
```

Comment:
Those two constructs would be simplified as well by using the `registerDecorators` proposal above.